### PR TITLE
refactor(SettingsAcountEdit): code

### LINKED ISSUE #386: settings-account-edit

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #386 

## Proposed Changes

-Change the imports to absolute:
-Move text to i18n
-Change the colors from the constants to similar ones in Color
-Transferring the screen logic to components
-Replace navigation hooks with typed navigation hooks in folder src/hooks

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-mobile/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: src/components/settings-account-edit.tsx [hunk 1]
```diff
@@ -57,7 +56,7 @@ export const SettingsAccountEdit = ({
           rightAction={
             inputName && (
               <IconButton onPress={cleanTextFile}>
-                <Icon s name="close_circle" color={LIGHT_GRAPHIC_BASE_2} />
+                <Icon i24 name="close_circle" color={LIGHT_GRAPHIC_BASE_2} />
```
Comment (gHashTag) [inline src/components/settings-account-edit.tsx:59]: Change color prop
```jsx
color={Color.graphicBase2}
```

File: src/components/settings-account-edit.tsx [hunk 2]
```diff
@@ -3,16 +3,15 @@ import React from 'react';
...
   Icon,
   IconButton,
   Input,
   KeyboardSafeArea,
-} from '../components/ui';
-import {I18N, getText} from '../i18n';
-import {LIGHT_GRAPHIC_BASE_2, LIGHT_GRAPHIC_GREEN_1} from '../variables';
+} from '@app/components/ui';
+import {I18N, getText} from '@app/i18n';
+import {LIGHT_GRAPHIC_BASE_2, LIGHT_GRAPHIC_GREEN_1} from '@app/variables';
```
Comment (gHashTag) [inline src/components/settings-account-edit.tsx:14]: Use `Color` enum for support dark theme

**Review by gHashTag — CHANGES_REQUESTED** [review]
Try to make the `<CustomHeader />` call the `getText` and `getColor` methods in itself.
As for example, `<Text/>` already does

File: src/components/settings-account-edit.tsx (hunk 2 above)
Comment (samandarkuchkarov) [reply src/components/settings-account-edit.tsx]: done

**Review by gHashTag — APPROVED** [review]

File: src/components/actions-sheet/index.android.tsx [hunk 3]
```diff
@@ -74,27 +70,27 @@ export const ActionsSheet = ({
...
+          styles.animateViewFade,
           slideFromBottomAnimation,
           {paddingBottom: bottom},
         ]}>
-        <View style={page.top}>
-          <Text t14 style={page.t8}>
+        <View style={styles.top}>
+          <Text t14 style={styles.t8}>
             {getText(I18N.actionSheetMessage)}
           </Text>
```
Comment (vivalaakam) [inline src/components/actions-sheet/index.android.tsx:83]:
```suggestion
          <Text t14 style={styles.t8} i18n={I18N.actionSheetMessage} />
```

File: src/components/actions-sheet/index.android.tsx [hunk 4]
```diff
@@ -74,27 +70,27 @@ export const ActionsSheet = ({
...
               {getText(I18N.actionSheetDiscard)}
             </Text>
           </TouchableOpacity>
         </View>
-        <View style={page.bottom}>
-          <TouchableOpacity style={page.margin} onPress={handleKeepEditing}>
-            <Text color={LIGHT_TEXT_BLUE_1} t14 style={page.tButton}>
+        <View style={styles.bottom}>
+          <TouchableOpacity style={styles.margin} onPress={handleKeepEditing}>
+            <Text color={LIGHT_TEXT_BLUE_1} t14 style={styles.tButton}>
```
Comment (vivalaakam) [inline src/components/actions-sheet/index.android.tsx:93]: i18n prop can be used

File: src/components/actions-sheet/index.android.tsx [hunk 5]
```diff
@@ -13,18 +13,14 @@ import Animated, {
 } from 'react-native-reanimated';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 
+import {Color} from '@app/colors';
+import {Text} from '@app/components/ui';
+import {I18N, getText} from '@app/i18n';
+import {LIGHT_TEXT_BLUE_1} from '@app/variables';
+
+import {createTheme} from './../../helpers/create-theme';
```
[the remark this answers is not in the document]
Comment (samandarkuchkarov) [reply src/components/actions-sheet/index.android.tsx]: done

File: src/components/actions-sheet/index.android.tsx [hunk 6]
```diff
@@ -74,27 +70,27 @@ export const ActionsSheet = ({
...
+        <View style={styles.top}>
+          <Text t14 style={styles.t8}>
             {getText(I18N.actionSheetMessage)}
           </Text>
-          <View style={page.line} />
-          <TouchableOpacity style={page.margin} onPress={handleDiscard}>
-            <Text color={DARK_GRAPHIC_RED_1} t14 style={page.tButton}>
+          <View style={styles.line} />
+          <TouchableOpacity style={styles.margin} onPress={handleDiscard}>
+            <Text color={Color.graphicRed1} t14 style={styles.tButton}>
```
[the remark this answers is not in the document]
Comment (samandarkuchkarov) [reply src/components/actions-sheet/index.android.tsx]: done

File: src/components/actions-sheet/index.android.tsx (hunk 4 above)
Comment (samandarkuchkarov) [reply src/components/actions-sheet/index.android.tsx]: done

File: src/components/actions-sheet/index.android.tsx (hunk 3 above)
Comment (samandarkuchkarov) [reply src/components/actions-sheet/index.android.tsx]: done

**Review by vivalaakam — APPROVED** [review]

**Pull request merged**